api/api.go

```diff
@@ -41,6 +41,8 @@ type Boost interface {
 	BoostDeal(ctx context.Context, dealUuid uuid.UUID) (*smtypes.ProviderDealState, error)                                                      //perm:admin
 	BoostDealBySignedProposalCid(ctx context.Context, proposalCid cid.Cid) (*smtypes.ProviderDealState, error)                                  //perm:admin
 	BoostDummyDeal(context.Context, smtypes.DealParams) (*ProviderDealRejectionInfo, error)                                                     //perm:admin
+	BoostIndexerAnnounceDealRemoved(ctx context.Context, pieceCid cid.Cid) (cid.Cid, error)                                                     //perm:admin
+	LegacyDealByProposalCid(ctx context.Context, propCid cid.Cid) (storagemarket.MinerDeal, error)                                              //perm:admin
```

Comment:
This is added under `Boost` MethodGroup, so maybe add `Boost` prefix?

Reply:
Updated. The reason to have it under Boost group is ensure compatibility after removing legacy markets.